prov/gni/src/gnix_sep.c

```diff
@@ -119,6 +119,11 @@ static int gnix_sep_tx_ctx(struct fid_ep *sep, int index,
 		(index >= sep_priv->info->ep_attr->tx_ctx_cnt))
 		return -FI_EINVAL;
 
+	if (attr && attr->op_flags & ~GNIX_EP_OP_FLAGS) {
```

Comment:
What do you think about warning here in the case where the op_flags are valid, but not what the sep was opened with?  Or would that be an error?